### ISSUE TITLE
Missing column warnings should include pattern names

### ISSUE THREAD

**Issue by matentzn** [INCATools/dosdp-tools#389]
When running a huge process like:

```
dosdp-tools generate --catalog=catalog-v001.xml --infile=../patterns/data/default/ --template=../patterns/dosdp-patterns --batch-patterns="abnormalSizeOfAnatomicalEntity abnormalAbsenceOfMolecularFunction abnormallyDelayedBiologicalProcessInLocation abnormallyIncreasedWidthOfAnatomicalEntity abnormalStabilityOfAnatomicalEntity abnormallyIncreasedLengthOfAnatomicalEntity abnormalMorphologyOfAnatomicalEntity abnormallyDecreasedLengthOfAnatomicalEntity abnormallyPrematureBiologicalProcess abnormallyIncreasedOccurrenceOfBiologicalProcessInLocation abnormalBiologicalProcessInLocation abnormallyIncreasedNumberOfCellularComponent abnormallyDecreasedWidthOfAnatomicalEntity abnormalAnatomicalEntity abnormallyDelayedBiologicalProcess abnormallyIncreasedSizeOfAnatomicalEntity abnormallyDecreasedSizeOfCellularComponent abnormallyDecreasedOccurrenceOfBiologicalProcessInLocation abnormallyIncreasedRateOfBiologicalProcess abnormallyIncreasedSizeOfCellularComponent abnormallyDecreasedNumberOfCellularComponent abnormalAbsenceOfBiologicalProcessInLocation abnormalColorOfAnatomicalEntity abnormallyArrestedBiologicalProcess abnormallyIncreasedRateOfMolecularFunction abnormalCellularComponentProcess abnormallyIncreasedOccurrenceOfBiologicalProcess abnormalBiologicalProcess abnormalAbsenceOfProteinLocalizationInCellularComponent abnormalMorphologyOfCellularComponent abnormallyIncreasedEfficacyOfProteinLocalizationInCellularComponent abnormallyDecreasedRateOfBiologicalProcess abnormalEfficacyOfProteinLocalizationInCellularComponent abnormallyIncreasedNumberOfAnatomicalEntity abnormallyDecreasedSizeOfAnatomicalEntity abnormalMolecularFunction abnormallyDecreasedRateOfMolecularFunction abnormallyDecreasedOccurrenceOfBiologicalProcess abnormalAbsenceOfBiologicalProcess abnormallyDecreasedNumberOfAnatomicalEntity abnormallyDecreasedEfficacyOfProteinLocalizationInCellularComponent abnormallyDecreasedMassOfAnatomicalEntity abnormallyPrematureBiologicalProcessInLocation" --ontology=ddpheno-edit.obo --obo-prefixes=true --restrict-axioms-to=logical --outfile=../patterns/data/default
```

The following is not very helpful:

```
2022.01.19 15:06:47:000 [zio-def...] [WARN ] org.monarchinitiative.dosdp.cli.Generate.run:40 - Input is missing column for pattern variable <anatomical_entity>
2022.01.19 15:06:48:000 [zio-def...] [WARN ] org.monarchinitiative.dosdp.cli.Generate.run:40 - Input is missing column for pattern variable <location>
2022.01.19 15:06:48:000 [zio-def...] [WARN ] org.monarchinitiative.dosdp.cli.Generate.run:40 - Input is missing column for pattern variable <location>
2022.01.19 15:06:48:000 [zio-def...] [WARN ] org.monarchinitiative.dosdp.cli.Generate.run:40 - Input is missing column for pattern variable <location>
2022.01.19 15:06:49:000 [zio-def...] [WARN ] org.monarchinitiative.dosdp.cli.Generate.run:40 - Input is missing column for pattern variable <location>
```

It will take a long time figuring out which input tables have wrong column names.. I also feel like something as severe as as missing column should result in a total failure..

**Comment by dosumis** [comment]
Might be a general issue with warnings and filenames - see #387

**Comment by balhoff** [comment]
Fixed by #400.
